content/en/cosign/public_deployment.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Public Staging Environment"
```

Comment:
Can we change this title to "Public Deployment"?

Reply:
changed